src/components/password/components.test-pw.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import Password, { PasswordProps } from ".";
+import Box from "../box";
+import CarbonProvider from "../carbon-provider/carbon-provider.component";
+
+export const SIZES = ["small", "medium", "large"] as const;
+export const VALIDATIONS = ["error", "warning", "info"] as const;
+
+const PasswordComponent = ({ onChange, ...props }: PasswordProps) => {
```

Comment:
If this has been copied over from password-test.stories.tsx it can probably be removed from there.

Reply:
Both the sizes and validation consts are still needed, as is the PasswordComponent